gossip/src/cluster_info.rs

```diff
@@ -1476,10 +1476,14 @@ impl ClusterInfo {
                 .pull
                 .build_crds_filters(thread_pool, &self.gossip.crds, MAX_BLOOM_SIZE)
         } else {
-            pulls.values().flatten().cloned().collect()
+            pulls
+                .iter()
+                .flat_map(|(_, filters)| filters)
+                .cloned()
+                .collect()
         };
         self.stats.pull_from_entrypoint_count.add_relaxed(1);
-        pulls.insert(entrypoint, filters);
+        pulls.push((entrypoint, filters));
```

Comment:
is this sending every single filter to the entrypoint address? While also sending individual filters to other nodes?

Reply:
It only does so when the entrypoint has not been observed through gossip yet: lines 1455-1470.
Which tends to happen only during startup.

Reply:
so the assumption (or maybe its been observed) is that not enough validators startup at once to overload the entrypoints?

Reply:
yeah, probably.
Though as soon as the node obtains some other contact-infos, some throttling is implemented not to overload entrypoints:
https://github.com/anza-xyz/agave/blob/16770cce8/gossip/src/cluster_info.rs#L1455